dali/test/python/test_pipeline.py

```diff
@@ -656,10 +656,9 @@ def __init__(self, batch_size, num_threads, device_id):
                                                 mean = [128., 128., 128.],
                                                 std = [1., 1., 1.])
             self.affine = ops.WarpAffine(device = "gpu",
-                                         matrix = [1.0, 0.8, 0.0, 0.0, 1.2, 0.0],
-                                         fill_value = 128,
-                                         interp_type = types.INTERP_LINEAR,
-                                         use_image_center = True)
+                                            matrix = [1.0, 0.8, -0.8*112, 0.0, 1.2, -0.2*112],
```

Comment:
can you fix the indentation here?

Reply:
If there's something more substantial, I'll do that, otherwise it'll go with rotate PR. Is that OK?